src/test/java/teammates/test/cases/ui/browsertests/InstructorHomePageUiTest.java

```diff
@@ -250,7 +250,8 @@ public void testRemindActions(){
         homePage.goToPreviousPage(InstructorHomePage.class);
         
         ______TS("remind action: OPEN feedback session - inner button");
-        
+
+        homePage.waitForAjaxLoaderGifToDisappear();
```

Comment:
There are a few other instances of goToPreviousPage() below, which may need the same wait.
And now that i think about it, maybe checking for the table presence is better, since there will be more ajax loading in the table (response rate) which we do not need to wait for.